src/components/Main/Schedule/Table.tsx

```diff
@@ -42,6 +48,7 @@ export const Table = (props: TableProps) => {
           <span className={classes.titleText}>{title}</span>
         </h3>
       )}
+      {access}
```

Comment:
this looks like a debug that should eventually be deleted

Reply:
oh no it's ok. this is for proof of concept that user has read or write access